### DISCUSSION TITLE
Track unused account for a private organisation

### DISCUSSION THREAD

**Discussion by philippart-s** [community/community#24890] [Apps, API and Webhooks]
<p>Hi everybody,</p>
<p>We have a private organization on <a href="http://github.com" rel="nofollow noopener">github.com</a>, thus our accounts are business account.</p>
<p>To avoid to paye for “phantom” accounts I want to know how to track unsed account on my organization.</p>
<p>I look to the github API (v3) but I am a little lost on how to do this.</p>
<p>Anybody have an idea ?</p>
<p>Thanks.</p>
<p>Stef</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/philippart-s">@philippart-s</a>,</p>
<p>Thank you for being here! This sample code here that might help give you a general idea of which accounts are active within your org:</p>
<p><a href="https://github.com/github/platform-samples/tree/master/api/ruby/find-inactive-members" rel="nofollow noopener">https://github.com/github/platform-samples/tree/master/api/ruby/find-inactive-members</a></p>
<p>I hope this helps!</p>
<p>Andrea</p>

**Comment by philippart-s** [comment]
<p>Thanks, we are trying to use it but wa are not ruby complient :).</p>
<p>If it’s solve our problem I go back ti flag the solution.</p>

**Comment by philippart-s** [comment]
<p>The script is perfect, thanks again it solve our problem.</p>

**Comment by gordanazivkovic25** [comment]
<p>Can you tell me why i cant open Emprowr site,where i work?It is sign:-Attention,Metamask/eth phishing detect.What i need to do?</p>
